views/email/email_partials/email_single_breach.hbs

```diff
@@ -4,9 +4,9 @@
       </td>
       <td valign="top" style="vertical-align: top; padding-top:10px;">
         <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; margin: 0px; font-weight: 600; font-size: 16px;">{{ Title }} Breach</p>
-        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Breach Date:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ BreachDate }}</span></p>
+        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Breach Date:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{prettyDate BreachDate }}</span></p>
         <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Accounts:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ PwnCount }}</span></p>
-        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Data:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ DataClasses }}</span></p>
+        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Data:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{breachDataClasses DataClasses }}</span></p>
```

Comment:
Does this need an `{{#if DataClasses}}` check, similar to ./views/partials/breach_listing.hbs?

https://github.com/mozilla/blurts-server/blob/bbc186aebbcbf91ab94ca0b0cfbac55e22a0d5a7/views/partials/breach_listing.hbs#L9-L11

Reply:
Good catch.